plugins/main/public/controllers/management/components/management/configuration/util-components/configuration-settings-header.js

```diff
@@ -29,38 +29,33 @@ class WzConfigurationSettingsHeader extends Component {
     super(props);
   }
   render() {
-    const {
-      title,
-      description,
-      help,
-      children
-    } = this.props;
+    const { title, description, help, children, info } = this.props;
     return (
       <Fragment>
-        <EuiFlexGroup alignItems="center">
+        <EuiFlexGroup alignItems='center'>
           <EuiFlexItem>
-            <EuiTitle size="s">
+            <EuiTitle size='s'>
               <h2>{title}</h2>
             </EuiTitle>
-            {description && <EuiText color="subdued">{description}</EuiText>}
+            {description && <EuiText color='subdued'>{description}</EuiText>}
           </EuiFlexItem>
           <EuiFlexItem grow={false}>
             <EuiFlexGroup
-              alignItems="center"
-              gutterSize="none"
-              justifyContent="flexEnd"
+              alignItems='center'
+              gutterSize='none'
+              justifyContent='flexEnd'
             >
-              {help && (
+              {(help || info) && (
```

Comment:
Beware that the `help` variable can be a falsy value and then the `WzHelpButtonPopover` expects an array. This could cause an exception.